### PR TITLE
fix(elasticsearch): apply patches to versioned libraries

### LINKED ISSUE #2073: No trace for elasticsearch7
My team's app has to work with multiple versions of elasticsearch, so we use the version-specific libraries. I saw in the [0.46.0 release](https://github.com/DataDog/dd-trace-py/releases/tag/v0.46.0) that y'all added support for `elasticsearch7` so I immediately upgraded and gave it a shot. Unfortunately, I'm not seeing any spans for elasticsearch in the trace.

### Which version of dd-trace-py are you using?
0.46.0

### Which version of the libraries are you using?
Here are the ES libs we're using:
```
elasticsearch              1.9.0
elasticsearch-dsl          0.0.11
elasticsearch7             7.11.0
elasticsearch7-dsl         7.2.1
```

### Tracer setup
This is at the top of our WSGI:

If it matters, we're running a django 1.8

```python
from ddtrace import config, patch_all

patch_all(httplib=True)
# header names are not case sensitive here
config.trace_headers([
    'cache-control',
    'expires',
    'host',
    'referer',
    'server',
    'user-agent',
])
config.django.http.trace_query_string = True
```

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
# Bug fix

## Description

#1629 changed elasticsearch to patch on import but did not include all of the versioned elasticsearch modules (eg. elasticsearch6, elasticsearch7) in the list of modules to patch on import on. This meant that if the base elasticsearch module was not imported then the patches would not be applied for elasticsearch1, ..., elasticsearch7.

## Fix

Add the versioned elasticsearch modules to the patch on import list so that the patches are applied.

Fixes #2073.

# Checklist
- [x] Regression test added; or
- [x] Description of manual testing performed and explanation is included in the code and/or PR.
- [x] Added to the correct milestone.

# Testing
- [x] Tests are run against all relevant library versions (or confirm that the relevant versions are already being tested in `tox.ini`)

### CODE REVIEW

**Comment by Kyle-Verhoog** [comment]
Regression test failure is `test_elasticsearch7`:

```
=================================== FAILURES ===================================
_____________________________ test_elasticsearch7 ______________________________
b"No traces found for token 'tests.contrib.elasticsearch.test_elasticsearch_multi.test_elasticsearch7'...
```

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/5941/workflows/2e9a2bc3-e6de-4117-b9b5-fe6d24bd92fb/jobs/536675/parallel-runs/2?filterBy=ALL

File: ddtrace/monkey.py [hunk 1]
```diff
@@ -87,7 +87,16 @@
...
     "botocore": ("botocore",),
-    "elasticsearch": ("elasticsearch",),
+    "elasticsearch": (
+        "elasticsearch",
+        "elasticsearch1",
+        "elasticsearch2",
+        "elasticsearch3",
+        "elasticsearch4",
+        "elasticsearch5",
+        "elasticearch6",
```
Comment (jd) [inline ddtrace/monkey.py:97]: typo I hope
```suggestion
        "elasticesarch6",
```

File: ddtrace/contrib/elasticsearch/patch.py [hunk 2]
```diff
@@ -118,14 +118,3 @@ def _perform_request(func, instance, args, kwargs):
             return result
 
     return _perform_request
-
-
-# Backwards compatibility for anyone who decided to import `ddtrace.contrib.elasticsearch.patch._perform_request`
```
Comment (jd) [inline ddtrace/contrib/elasticsearch/patch.py:123]: ok though looks unrelated right?

**Review by jd — CHANGES_REQUESTED** [review]

File: ddtrace/monkey.py (hunk 1 above)
Comment (Kyle-Verhoog) [reply ddtrace/monkey.py:97]: Makes a good case for having tests for each version! 🤦‍♂️

File: ddtrace/contrib/elasticsearch/patch.py (hunk 2 above)
Comment (Kyle-Verhoog) [reply ddtrace/contrib/elasticsearch/patch.py:123]: I did it already in #2141

File: ddtrace/monkey.py (hunk 1 above)
Comment (Kyle-Verhoog) [reply ddtrace/monkey.py]: done now

**Review by P403n1x87 — APPROVED** [review]

**Review by jd — DISMISSED** [review]

**Comment by Kyle-Verhoog** [comment]
Hmm the tests are failing because some `sqlite` spans are being sent during the test cases 🤔 might be related to junit

**Review by P403n1x87 — APPROVED** [review]

**Review by jd — APPROVED** [review]
